core/parameters/res/runtime_configs/parameters.snap

```diff
@@ -147,7 +147,7 @@ wasm_alt_bn128_g1_sum_element                  5_000_000_000
 wasm_yield_create_base                       153_411_779_276
 wasm_yield_create_byte                            15_643_988
 wasm_yield_resume_base                     1_195_627_285_210
-wasm_yield_resume_byte                            47_683_715
+wasm_yield_resume_byte                     1_195_627_285_210
```

Comment:
Is this right? Perhaps I don't understand what parameters.snap is supposed to do.

Reply:
So, this is a snapshot of the parameters that we have at the current stable protocol version. I hadn't updated the protocol version in the previous revision, but I have done so now so the snapshot has been updated.